File: Modul-6/Soal-1/PRAK601-2410817110016-MuhammadAzriRahman.py
baris, kolom = map(int, input().split())

print("Input")
elemen = list(map(int, input().split()))

print("Output")
for i in range(baris):
    for j in range(kolom):
        print(elemen[i * kolom + j], end=" ")
    print()

File: Modul-6/Soal-1/PRAK601-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

int main() {
    int baris, kolom;

    scanf("%d %d", &baris, &kolom);

    int array[baris][kolom];

    printf("Input\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &array[i][j]);
        }
    }
    printf("Output\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", array[i][j]);
        }
        printf("\n");
    }

    return 0;
}
